data.tf

```diff
@@ -0,0 +1,5 @@
+data "azurerm_key_vault_secret" "idamUserDisposerAlertEmail" {
```

Comment:
You don't need idamUserDisposerAlertEmail if slack webhook url is already provided